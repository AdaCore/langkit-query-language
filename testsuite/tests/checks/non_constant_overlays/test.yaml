driver: 'checker'
rule_name: Non_Constant_Overlays
input_sources: ['overlay.adb']
