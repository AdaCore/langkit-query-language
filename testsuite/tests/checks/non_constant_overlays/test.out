overlay.adb:5:4: error: constant object overlays a non-constant object
5 |    for C'Address use V'Address;    --  FLAG
  |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

overlay.adb:9:4: error: constant object overlays a non-constant object
9 |    for V'Address use C'Address;           --  FLAG: C is not Volatile
  |    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

